### PR TITLE
update gitlab-ce to 9.4.2 and gitlab-runner to 9.4.1

### PR DESCRIPTION
**Pull request by wtayyeb**
update gitlab-ce to 9.4.2 and gitlab-runner to 9.4.1

### CODE REVIEW

**Comment by rawmind0** [comment]
Hi @wtayyeb ... thanks for your PR....

Could you please add new files to ```templates/gitlab-multi-runner/1``` and ```templates/gitlab/1``` instead of modifying   ```templates/gitlab-multi-runner/0``` and ```templates/gitlab/0``` files??

File: templates/gitlab-multi-runner/1/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - docker
+    - --description
+    - Rancher Docker Runner
+    - --docker-image
+    - docker:latest
+    - --docker-volumes
+    - /var/run/docker.sock:/var/run/docker.sock
+    - --docker-privileged
+    labels:
+      io.rancher.container.pull_image: always
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/1/docker-compose.yml]: Specific version don't need to pull always...
```
-      io.rancher.container.pull_image: always
```

File: templates/gitlab-multi-runner/1/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    volumes:
+    - /var/run/docker.sock:/var/run/docker.sock
+    tty: true
+    volumes_from:
+    - gitlab-runner-config
+    command:
+    - run
+    labels:
+      io.rancher.sidekicks: gitlab-runner-config
+      io.rancher.container.pull_image: always
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/1/docker-compose.yml]: Specific version don't need to pull always...
```
-      io.rancher.container.pull_image: always
```

File: templates/gitlab-multi-runner/1/rancher-compose.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - variable: "GITLAB_TAGS"
+      label: "Gitlab Tags"
+      description: "Tags to apply"
+      type: "string"
+      default: "dev"
+      required: false
+
+services:
+  gitlab-runner-config:
+    scale: 1
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/1/rancher-compose.yml]: If you are using global scheduler, don't need to define scale.
```
-    scale: 1
```

File: templates/gitlab-multi-runner/1/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+      io.rancher.scheduler.global: 'true'
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/1/docker-compose.yml]: If scheduler is global, please, use optional host_label to control where to run the service, if you like to, using go tempalting. 
```
+    {{- if ne .Values.host_label ""}}
+      io.rancher.scheduler.affinity:host_label: ${host_label}
+    {{- end}}
```

File: templates/gitlab-multi-runner/1/rancher-compose.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      type: "string"
+      default: "xxxxxxxxxxxxxxxxxxxx"
+      required: true
+
+    - variable: "GITLAB_TAGS"
+      label: "Gitlab Tags"
+      description: "Tags to apply"
+      type: "string"
+      default: "dev"
+      required: false
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/1/rancher-compose.yml]: Add host_label question...
```
+    - variable: host_label
+      label: "Host with Label to deploy gitlab-runner on"
+      description: |
+        Host label to use as gitlab-runner 'value' tag.
+        Example: 'gitlab-runner=true'
+      required: false
+      default: ""
+      type: "string"
```

File: templates/gitlab-multi-runner/1/docker-compose.yml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+    - --description
+    - Rancher Docker Runner
+    - --docker-image
+    - docker:latest
+    - --docker-volumes
+    - /var/run/docker.sock:/var/run/docker.sock
+    - --docker-privileged
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/1/docker-compose.yml]: If scheduler is global, please, use optional host_label to control where to run the service, if you like to, using go tempalting. 
```
+      io.rancher.scheduler.global: 'true'
+    {{- if ne .Values.host_label ""}}
+      io.rancher.scheduler.affinity:host_label: ${host_label}
+    {{- end}}
```

**Comment by rawmind0** [comment]
Thanks @wtayyeb ...

Could you please to take a look at the change request? Just adding optional host_label to control where to deploy global scheduling service.

**Comment by wtayyeb** [comment]
@rawmind0 OK. I applied your comments. please review it again.

**Comment by rawmind0** [comment]
Thanks @wtayyeb ...

LGTM..

**Pull request merged**